patterns/vsi-quickstart/variables.tf

```diff
@@ -40,6 +40,14 @@ variable "resource_tags" {
   default     = []
 }
 
+variable "user_data" {
```

Comment:
This will not work well in quickstart, as we are hard-coding (in the override) the VSI configurations. If we still want to support user_data in quickstart, we should redesign this pattern a little bit so that user supplied user_data can be inserted into the override json. This does mean a bit of a redesign of this pattern where we would move the override from an input variable with default, and instead have it in the main.tf with variable placeholders. See the `roks-quickstart/main.tf` for an example.